tests/baselines/reference/parserRealSource7.errors.txt

```diff
@@ -658,7 +658,7 @@ tests/cases/conformance/parser/ecmascript5/parserRealSource7.ts(828,13): error T
 !!! error TS2304: Cannot find name 'StringHashTable'.
                     modType = new ModuleType(enclosedTypes, ambientEnclosedTypes);
                                   ~~~~~~~~~~
-!!! error TS2304: Cannot find name 'ModuleType'.
+!!! error TS2552: Cannot find name 'ModuleType'. Did you mean 'moduleDecl'?

```

Comment:
Since right now we're calculating edit distance in a case-insensitive way - the edit distance is 4, which is the maximum allowable distance for a string of length 10 (I didn't change that heuristic). TBQH I think this one _should_ have appeared before I changed the underlying algorithm. I also think there's value in including the case in the distance heuristic - it's just more information.

In any case, since transpositions are cheaper, it's probably safe to lower the heuristic for maximum edit distance.